### DISCUSSION TITLE
Pay-as-you-go Integration

### DISCUSSION THREAD

**Discussion by mprince2k18** [shipfastlabs/larasonic-vue#9] [Roadmap]
I believe incorporating a Pay-as-you-go feature into this project would be a valuable enhancement. For instance, it can benefit users who need temporary access to resources, such as scaling up their usage during peak demand periods or running one-time tasks without committing to a long-term plan. This flexibility can attract a broader range of users with diverse needs.

**Comment by pushpak1300** [comment]
Sur that can be on roadmap as well. 
Let me first get the subscription feature live and I can definitely work on this.
